support-frontend/assets/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -189,11 +189,7 @@ const onComplete = (res: PaymentResult) => {
 };
 
 function updateTotal(props: PropTypes) {
-	// When the other tab is clicked, the value of amount is NaN
-	if (
-		!Number.isNaN(props.amount) &&
-		props.paymentRequestObject.status === 'AVAILABLE'
-	) {
+	if (props.paymentRequestObject.status === 'AVAILABLE') {
```

Comment:
I've simplified this check as now we don't call `updateTotal` until after we've checked that the amount is valid